charts/system/intel-gpu-plugin/values.yaml

```
@@ -0,0 +1,53 @@
+image:
+  repository: intel/intel-gpu-plugin
+  pullPolicy: IfNotPresent
+  tag: 0.30.0@sha256:e96c9955e5aced52f215f7eff77f679c4c400a16a42b1f2c0e8ed9ba76cae3c4
+
+service:
+  main:
+    enabled: false
+    ports:
+      main:
+        enabled: false
+
+workload:
+  main:
+    podSpec:
+      containers:
```

```suggestion
      type: DaemonSet
      containers:
```

We should also probable have some kind of node selector too